.gitignore

```diff
@@ -1,6 +1,7 @@
 .vscode
 demo*
 node_modules
+pnpm-lock.yaml
```

Comment:
This repo is currenly using npm, please don't add pnpm related changes in this PR. However if you'd like to migrate this project to pnpm I'd be greatful for a seprate PR ;)